docs/source/api_reference.rst

```diff
@@ -156,6 +156,11 @@ Cumulative Transform Primitives
 
     Diff
     TimeSincePrevious
```

Comment:
Do the above two belong under "Cumulative Transform Primitives"?

Reply:
that's the best place to put them for now since they do use multiple rows of the dataframe, although in this case just the previous